friends/src/Components/Friend.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+
+class Friend extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      friend: null
+    };
+  }
+
+  render() {
+    const { name, age, email, friends } = this.state.friend;
```

Comment:
The state is null. In order to get destruct each friend, you can do so from the props.
` const { name, age, email } = this.props.friend;`
The friends array is passed down as props "friend" on line 33, in the *FriendsList* component.